erizoAPI/MediaStream.cc

```diff
@@ -117,6 +117,13 @@ std::string MediaStream::toLog() {
   return "id: " + id_;
 }
 
+void MediaStream::computePromiseTimes(erizo::time_point scheduled_at,
+    erizo::time_point started, erizo::time_point end) {
+  promise_durations_.add(started - scheduled_at);
```

Comment:
I think this is inverted compared to what we do with tasks. Shouldn't the  `delay = started - scheduled_at`, meaning the delay before the promise starts and  `duration = end - started` meaning the duration of the actual promise?

Reply:
It is not inverted, but that's true that I should use a better naming instead of started/scheduled_at/ended, and I'll update the description for the Prometheus metrics.